README.md

```diff
@@ -15,7 +15,7 @@ If you have a question about the Istio Manager or have a problem using it, pleas
 
 We are using [Bazel 0.4.3](https://bazel.io) to build Istio Manager:
 
-    bazel build //...
+    bazel build //cmd/...
```

Comment:
ok. Maybe we can create an issue to make sure we have a better solution in future.